packages/server/__snapshots__/task_not_registered_spec.coffee

```diff
@@ -0,0 +1,59 @@
+exports['e2e task fails 1'] = `
+Started video recording: /foo/bar/.projects/task-not-registered/cypress/videos/abc123.mp4
+
+  (Tests Starting)
+
+
+  1) fails because the 'task' event is not registered in plugins file
+
+  0 passing
+  1 failing
+
+  1)  fails because the 'task' event is not registered in plugins file:
+     CypressError: cy.task('some:task') failed with the following error:
+
+> "The 'task' event has not been registered in the plugins file, so cy.task() cannot run"
```

Comment:
Not sure we want to add a stack trace here because it's strictly internal to the task implementation. There's nothing actionable for the user to do here I don't think.

Instead I suggest we link them to the `cy.task` API docs since that will help them explain what to do...

`https://on.cypress.io/api/task`